tekton/tasks/binary-container-cachi2.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: binary-container-cachi2-0-1  # dot is not allowed in the name
+spec:
+  description: >-
+    OSBS cachi2 task for building binary container image
+  params:
+    - name: osbs-image
+      description: The location of the OSBS builder image (FQDN pullspec)
+      type: string
+    - name: pipeline-run-name
+      type: string
+      description: PipelineRun name to reference current PipelineRun
+    - name: user-params
+      type: string
+      description: User parameters in JSON format
+
+  workspaces:
+    - name: ws-build-dir
+    - name: ws-context-dir
+    - name: ws-home-dir
+    - name: ws-registries-secret  # access with $(workspaces.ws-registries-secret.path)/token
+    - name: ws-koji-secret  # access with $(workspaces.ws-koji-secret.path)/token
+    - name: ws-reactor-config-map
+    - name: ws-autobot-keytab
+
+  stepTemplate:
+    env:
+      - name: HOME
+        value: $(workspaces.ws-home-dir.path)
+
+  steps:
+    - name: binary-container-cachi2-init
+      image: $(params.osbs-image)
+      workingDir: $(workspaces.ws-home-dir.path)
+      resources:
+        requests:
+          memory: 512Mi
+          cpu: 250m
+        limits:
+          memory: 1Gi
+          cpu: 395m
+      script: |
+        set -x
+        atomic-reactor -v task \
+          --user-params="$(params.user-params)" \
```

Comment:
Is this how we pass user params in other tasks? It would surprise me if this didn't break, it's JSON and contains double-quotes (and `$(params)` substitution doesn't escape any special characters)

It would be safer to pass it as an environment variable

Reply:
I tested this bash magic locally and works fine, it works fine also in integration tests; :magic_wand:

Reply:
updated to env var